src/elements/icon-tile/src/index.tsx

```diff
@@ -6,17 +6,23 @@ import { jsx, Styled } from 'theme-ui'
 interface Props extends React.HTMLAttributes<any> {
   icon: React.ReactNode
   className?: string
+  alignVertically?: boolean
 }
-const IconTile: React.FC<Props> = ({ icon, children, className }) => {
+const IconTile: React.FC<Props> = ({
+  icon,
+  children,
+  className,
+  alignVertically
```

Comment:
can this be a variant of type string, default value horizontal with option to pass in vertical

Reply:
use an Enum for the 2 values

Reply:
Updated